src/emulator/auth/state.ts

```diff
@@ -631,6 +692,34 @@ export class AgentProjectState extends ProjectState {
     return true;
   }
 
+  get config() {
+    return this._config;
+  }
+
+  get blockingFunctionsConfig() {
+    return this._config.blockingFunctions;
+  }
+
+  set blockingFunctionsConfig(blockingFunctions: BlockingFunctionsConfig) {
+    this._config.blockingFunctions = blockingFunctions;
+  }
+
+  // TODO(lisajian): Once v2 API discovery is updated, type of update should be
+  // changed to Schemas["GoogleCloudIdentitytoolkitAdminV2Config"].
+  updateConfig(
+    update: Schemas["GoogleCloudIdentitytoolkitAdminV2Config"] & { usageMode?: UsageMode },
+    updateMask: string | undefined
+  ): Config {
+    // Empty masks indicate a full update.
+    if (!updateMask) {
+      this.oneAccountPerEmail = !update.signIn?.allowDuplicateEmails ?? true;
+      this.blockingFunctionsConfig = update.blockingFunctions ?? {};
```

Comment:
This needs to be validated in `operations.ts`

Reply:
We just need to validate the strings are well-formatted URIs with `http` or `https` scheme. Otherwise, we'll have a hard time when we try to make requests later.

Reply:
Ohh, got it, thanks for the clarification! Updated with some error handling of blocking functions!